src/app/common/confirmation-modal/confirmation-modal.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 mat-dialog-title>{{data.headerText}}</h1>
+<div mat-dialog-content>{{data.confirmationText}}</div>
+<div mat-dialog-actions>
+    <button mat-button
+            [mat-dialog-close]="false">{{data.denyBtnText || "No"}}</button>
```

Comment:
Please unify code -> empty space after/before `{`/`}` and use single quotes for strings in the interpolation.